OJP/OJP_FareSupport.xsd

```diff
@@ -697,15 +697,20 @@
 	</xs:group>
 	<xs:complexType name="TripFareResultStructure">
 		<xs:annotation>
-			<xs:documentation>Structure of a Single Trip Fare Request result</xs:documentation>
+			<xs:documentation>Structure of a single TripFareResult. This may cover the whole trip or only the part the service can answer. If only partial answers can be delivered, then this must be indicated in the error condition (as a warning). For a given trip multiple FareResults may need to be combined to get the whole tariff. A typical example may be classic public transport and a sharing leg. Some legs, e.g. TransferLegs and walking ContinuousLeg will never have a price. The service may include them in one of the tickets. If the whole trip is for free, then a TripFareResult with a price of 0 is expected. It is possible to have combined FareResults (e.g. for a bundle) and in addition price each leg individually.</xs:documentation>
```

Comment:
```suggestion
			<xs:documentation>Structure of a single TripFareResult. This may cover the whole trip or only the part the service can answer. If only partial answers can be delivered, then this must be indicated with FARE_ADDITIONALTICKETS (as a warning). For a given trip multiple FareResults may need to be combined to get the whole tariff. A typical example may be classic public transport and a sharing leg. Some legs, e.g. TransferLegs and walking ContinuousLeg will never have a price. The service may include them in one of the tickets. If the whole trip is for free, then a TripFareResult with a price of 0 is expected. It is possible to have combined FareResults (e.g. for a bundle) and in addition price each leg individually.</xs:documentation>
```